packages/web-api/src/types/request/apps.ts

```diff
@@ -1,8 +1,9 @@
 import type { AppID, CursorPaginationEnabled, TokenOverridable, OAuthCredentials } from './common';
 import type { Manifest } from './manifest';
+import { OptionalArgument } from '../helpers';
 
 // https://api.slack.com/methods/apps.connections.open
-export interface AppsConnectionsOpenArguments { }
+export type AppsConnectionsOpenArguments = OptionalArgument<object>;
```

Comment:
⚠️ **BREAKING CHANGE** (technically, not pragmatically)

`OpenIDConnectUserInfoArguments` & `AppsConnectionsOpenArgument` are declared as empty interfaces. 

An empty interface allows for any non nullable value, not an object with no properties. It is well [explained in `typescript-eslint` rule `no-empty-interface`](https://typescript-eslint.io/rules/no-empty-interface/). In order to specify an empty object, [`object` can be used](https://www.typescriptlang.org/docs/handbook/2/functions.html#object). Using that then

This would be a breaking change if someone is using a non-nullable and non-object value as argument to one of those methods. Like `apps.connections.open(42)`. But that shouldn't have worked in the first place

Reply:
Can use `{}` and ignore `ESLint` rule complaining about it if we want to be sure no breaking change happens (could be done in next major as cleanup?)

Reply:
Huh, TIL! I think as long as the previous "expected" usage of these methods (i.e. needing to provide an empty object `{}` as a parameter for these methods that accept optional arguments) are a) enshrined in tests and b) pass, then that's all that matters to me.

Reply:
Same here tbh! Didn't know about that either, it was ESLint rule complaining that lead me through the rabbit hole.

Noice then 👍